src/message/__tests__/fetchActions-test.js

```diff
@@ -334,9 +334,8 @@ describe('fetchActions', () => {
       expect(actions.length).toBeGreaterThanOrEqual(1);
       const [action] = actions;
       expect(action.type).toBe('MESSAGE_FETCH_START');
-      if (action.type === 'MESSAGE_FETCH_START') {
-        expect(action.numBefore).not.toBeGreaterThan(0);
-      }
+      invariant(action.type === 'MESSAGE_FETCH_START', 'expect failed');
```

Comment:
Nice! Yeah, I felt weird adding these conditionals. But less weird than adding `$FlowFixMe`s. Glad to have this solution.